docs/project/changelog.md

```diff
@@ -22,6 +22,9 @@ myst:
 - {{ Fix }} Fixed `LONG_BIT definition appears wrong for platform` error happened in out-of-tree build.
   {pr}`4136`
 
+- {{ Enhancement }} Unvendored hiwire
```

Comment:
Maybe this is too vague. It is not a user-facing change so maybe we need to

1) remove it from the change log at all
2) or add more detailed information what happend

Reply:
Agreed, maybe just remove it... maybe we should have a change log tag like `architecture` for non user facing stuff like this but I'm okay leaving it out.

Reply:
Removing this for now, we can revisit if these changes deserve a changelog entry later.